cli/root.go

```diff
@@ -1,6 +1,9 @@
 package cli
 
-import "github.com/spf13/cobra"
+import (
+	"github.com/spf13/cobra"
+	"time"
+)
```

Comment:
Please format the source files using [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports).